web/build.gradle

```diff
@@ -17,13 +17,18 @@ bootJar {
 }
 sourceCompatibility = '11'
 
-repositories { mavenCentral() }
+repositories { 
+    mavenLocal()
+    mavenCentral()
+    maven { url 'https://jitpack.io' }
+}
 
 dependencies {
     compile 'org.springframework.boot:spring-boot-starter-web'
     compile 'org.springframework.boot:spring-boot-starter-data-rest'
     compile project(':data')
     testCompile 'org.springframework.boot:spring-boot-starter-test'
+    implementation 'com.github.papaya-agh:papaya-interface:IO2019TEAM-45-SNAPSHOT'
```

Comment:
Still - przed mergem to będzie do zmiany, bo na masterze musi być odnośnik na mastera

Reply:
Więc na interfejsach musi pójść merge pierwszy

Reply:
Bo inaczej CI, którego jeszcze nie mamy, będzie smutne